tests/testthat/test-root.R

```diff
@@ -178,6 +178,23 @@ test_that("is_git_root", {
                  "No root directory found.* a directory `.*`"),
     TRUE
   )
+
```

Comment:
Would you mind moving this to a separate test? Duplicate the setup if necessary, or extract it to a function.